packages/ui/styles/theme.css

```diff
@@ -93,3 +93,14 @@
   mask-composite: exclude;
   -webkit-mask-composite: xor;
 }
+
+.hide-scrollbar::-webkit-scrollbar {
+      display: none;
+  }
+
+.hide-scrollbar {
+    /* IE and Edge */
+    -ms-overflow-style: none;
+    /* Firefox */
+    scrollbar-width: none;
+}
```

Comment:
@Mythie I wasn't sure if webkit css rules are allowed for inline in css, so added them here. If you give an ack regarding inline ones, will add it through tailwind classes :)

and if so, I will just add `[&::-webkit-scrollbar]:hidden` to the classes